Issue 5: convert dates to ISO-8601 format

wdduncan opened the pull request:
See #5

dwinston commented:
This still needs code added, right? Or is the PR intended to simply be an addition of example json files?

wdduncan commented:
@dwinston I've added the code to change the `add_date` and `mod_date` fields to iso-8601 format. I've also added the `nmdc_database.json.zip` that is the output of the `make run-etl` target. I wasn't sure if needed to re-run `make run-elt` to incorporate the new data or if you could slurp in the zip file.
